app/views/DR/Maps/Map.js

```diff
@@ -24,6 +24,7 @@ import {
   requestCovid19Hospitals,
   requestCovid19Laboratories,
 } from '../../../helpers/Request';
+import languages from '../../../locales/languages';
```

Comment:
Please use the hook **useTranslation** in that way we can use the same instance of the language and the label will change at the same moment of the others
add this in the function `const { t } = useTranslation();`
```suggestion
import { useTranslation } from 'react-i18next';
```